src/styles/blocks/header.scss

```diff
@@ -0,0 +1,66 @@
+.header {
+  height: $header-height;
+
+  display: flex;
+  align-items: stretch;
+  justify-content: space-between;
+  padding-right: 50px;
+}
+
+.logo {
+  display: inline-block;
+  align-self: center;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
+}
+
+.nav {
+  &__link {
+    display: block;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: $color__navigation__link;
+    font-weight: 500;
+
+    transition-duration: $hover-animation-duration;
+
+    &:hover {
+      color: $color__active-link;
+    }
+  }
+
+  &__list {
+    display: flex;
+    height: 100%;
+    margin-block: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__item {
+    display: flex;
+    align-items: center;
+    list-style: none;
+    position: relative;
+
+    &:not(:first-child) {
+      margin-left: 20px;
+    }
+  }
+}
+
+.is-active {
+  color: $color__active-link;
+}
+
+.is-active::after {
+  content: '';
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  height: 4px;
+  background-color: $color__active-link;
+  border-radius: 8px;
+}
```

Comment:
1. Use nesting
2. Are you sure you need it as a block?

Reply:
yes, i need to have exectly class "is-active" for tests ( So i just moved it to separate file.

Reply:
> yes, i need to have exectly class "is-active" for tests ( So i just moved it to separate file.

Agree here - let's keep it